### ISSUE TITLE
What should we do with the sigstore files?

### ISSUE THREAD

**Issue by ocelotl** [open-telemetry/sig-security#53]
I managed to use a github action to include the sigstore file in a release:

https://github.com/SecuringCarter/opentelemetry-python/releases/tag/6

Is this what we want to do with the sigstore files?

**Comment by ocelotl** [comment]
@jpkrohling

**Comment by jpkrohling** [comment]
Not quite: checking the Python docs, I would expect the sigstore file to be uncompressed under the released artifacts. Do you provide any archive with the artifacts being released? Or are those only the ones provided automatically by GitHub?

**Comment by jpkrohling** [comment]
I see now that you are signing only one file, `opentelemetry-api/pyproject.toml`. Would that be sufficient to reassure your users? Python itself is signed with sigstore as well, but they distribute a tgz file and sign that tgz.
